exo-go/src/process_helpers/index.go

```diff
@@ -38,6 +40,17 @@ func Start(dir string, commandWords ...string) (*exec.Cmd, io.WriteCloser, *byte
 	return cmd, in, &out, nil
 }
 
+// Wait waits for the given text for the specified duration
+func Wait(stdout fmt.Stringer, text string, done func()) {
```

Comment:
I think it would be better if we use pipes here. If we have the commands stdout pipe, I believe we can read from it and that will block until data arrives. Thus we shouldn't need to poll and can return right when the data comes in.